MovieNight/ActorViewController.swift

```diff
@@ -57,6 +57,10 @@ class ActorViewController: UIViewController, UITableViewDataSource, UITableViewD
         return cell
     }
     
+    func shouldFetchNextPage(indexPath: IndexPath) -> Bool {
+        return (actors.count - indexPath.row) == 5 && hasNextPage ? true : false
```

Comment:
You don't need the `? true : false`.  You can leave it as 

`return (actors.count - indexPath.row) == 5 && hasNextPage`